cmd/create/cluster/cmd.go

```diff
@@ -2020,6 +2198,27 @@ func run(cmd *cobra.Command, _ []string) {
 		isHostedCP,
 		multiAZ)
 
+	var isClusterAutoscalerSet bool
```

Comment:
Should this be considered true by default if there are changes to any of the autoscaler params from the args?

Reply:
I changed this to pull out all the variables that check if the flags got changed and use that to determine if autoscaler is set. Let me know if this implementation does not work.